docs/wiki/Contribution guide - Contribution flow.md

```diff
@@ -53,7 +51,6 @@ How you proceed from here, depends on your scenario:
 - In case your contribution involves changes to the library (the modules) and/or to the CI environment (the pipelines), we kindly ask you to first review the [Module design](./The%20library%20-%20Module%20design) and/or the [Pipeline design](./The%20CI%20environment%20-%20Pipeline%20design) sections respectively, to make sure your contribution complies with the repository's design and principles.
 
 - If your contribution involves the creation of a new module or an update to an existing module:
-  - Make sure you also reflect your changes in the module's `README.md` file. Refer to the [Generate module Readme](./Contribution%20guide%20-%20Generate%20module%20Readme) section for details on how to leverage the Readme generator utility to support you with this task.
```

Comment:
Is it intended to remove this step completely?

Reply:
No. Just added it again.